tests/compilerfeatures/trepl.nim

```diff
@@ -0,0 +1,54 @@
+discard """
+  description: "Ensure that the basic REPL functionality works"
+  targets: native
```

Comment:
Should a timeout be added because this test is susceptible to bugs causing it to get stuck on reads?

Reply:
Good call, I'll add a timeout. It currently won't protect against the test getting stuck, however, since testaement's `timeout` feature is only an "after the fact" check.